client/src/plus/common/index.tsx

```diff
@@ -31,3 +32,22 @@ export function DataError({ error }: { error: Error }) {
     </NoteCard>
   );
 }
+
+export function _getIconLabel(url) {
+  let category = docCategory({ pathname: url });
+
+  if (category) {
```

Comment:
Matter of taste, but I prefer to return early:

```suggestion
  if (!category) {
    return "docs":
  }
```

Makes the function easier to skim.
```